### PR TITLE
Migration to SQLAlchemy

### PR DESCRIPTION
**Pull request by alexfirilov**
Updated the backend microservices to utilize SQLAlchemy instead of direct SQL connections, due to issues with the user and order backend services.

### CODE REVIEW

File: order-service/app.py [hunk 1]
```diff
@@ -1,76 +1,81 @@
...
+            })
+        else:
+            return jsonify({'message': 'Order not found'}), 404
+    finally:
+        session.close()
 
 if __name__ == '__main__':
-    app.run()
\ No newline at end of file
+    app.run(host='0.0.0.0', port=5000)
```
Comment (noambenm) [inline order-service/app.py]: Remove the args, they are declared in the docker file

File: product-service/app.py [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
+            'description': p.description,
+            'price': str(p.price)
+        } for p in products])
+    finally:
+        session.close()
 
 if __name__ == '__main__':
-    app.run()
\ No newline at end of file
+    app.run(host='0.0.0.0', port=5000)
```
Comment (noambenm) [inline product-service/app.py]: Remove the args, they are declared in the docker file

**Review by noambenm — CHANGES_REQUESTED** [review]

File: order-service/app.py (hunk 1 above)
Comment (alexfirilov) [reply order-service/app.py]: Okay

File: product-service/app.py (hunk 2 above)
Comment (alexfirilov) [reply product-service/app.py]: Fuck you, eat shit and die

**Comment by alexfirilov** [comment]
New commit - 7ca21fbaa30cb28311498edf4829ae1f2ce5ce52
addressed those requests.

**Review by noambenm — APPROVED** [review]

**Pull request merged**